### DISCUSSION TITLE
How to use react-contexify with react-treebeard !?

### DISCUSSION THREAD

**Discussion by Ridz0o0o** [fkhadra/react-contexify#253] [Q&A]
Hi Geeks,
Firstly, I just came across this library and looks like it's really an awesome one.

So, I am using react-treebeard in my project and I wonder how can I integrate "react-contexify" with it?
I want right-click menu on click of specific nodes of the tree only.

Could someone please help me with that? 
@fkhadra , thoughts on this if it's possible ?

Many Thanks

**Comment by meticulousfan** [comment]
Hi.
Those are my opinions. I hope it would be helpful for you.
```
import { Treebeard } from 'react-treebeard';
import { ContextMenuProvider, Menu, Item } from 'react-contexify';

const MyComponent = () => (
  <div>
    <ContextMenuProvider id="context-menu">
      <Treebeard data={treeData} />
    </ContextMenuProvider>
    <MyContextMenu />
  </div>
);

const MyContextMenu = () => (
  <Menu id="context-menu">
    <Item>Menu Item 1</Item>
    <Item>Menu Item 2</Item>
    <Item>Menu Item 3</Item>
  </Menu>
);
```
